content/organization/tech/accounts/index.en.md

```diff
@@ -82,13 +82,17 @@ A good strategy might be to make your group private and ask candidates questions
 
 ## E-Mail
 
-* You can get a chapter email (yourcity\@rladies.org) which can be set up to forward to all the organisers of a chapter.  As of March 18, 2019, we are no longer issuing personal rladies.org emails (yourname\`@rladies.org) to organizers and will have chapter emails only.
-   * Send an email (`email@rladies.org`) and volunteers in charge will set up your chapter email for you.  Please be patient, as these requests are processed weekly.
-   * In the email, please send the personal email addresses for you and your co-organizers if you would like chapter email to forward there, otherwise, please be responsible for checking your chapter email via webmail or by setting it up on your email client.
-* 🚨 **[VERY IMPORTANT]** **Your chapter email account should be used to set up any chapter social media accounts (e.g. Twitter, Instagram, etc).**
-* Find instructions on [how use your rladies.org email in a mail client or within Gmail](/organization/email/).
-* If you have an issue with emails send an email to `email@rladies.org` or try to crowd-source a solution in the `#email_etc_support` channel of the organizers slack.`
-
+* You can get a chapter email address (`yourcity@rladies.org`) which can be set up to forward to all the organisers of a chapter.  As of March 18, 2019, we are no longer issuing personal rladies.org emails (`yourname@rladies.org`) to organizers and will have chapter emails only. 
+* Starting in 2021, new chapter emails are just forwarding addresses (chapters started before this have full email accounts, but due to the overhead and changes by our hosting provider, new emails addresses can only be a regular email -or- a forwarding email, not both). With a forwarding email address, there’s no inbox or password, just a public facing email address that forwards to your local organiser team. 
```

Comment:
1. Could the text between parentheses be a footnote?
2. The text in parentheses is a bit misleading as it makes it sound possible to have a regular email.
3. Maybe add a tip e.g. when you answer emails to the chapter address, put the chapter address in copy?

Reply:
1. Sure.
2. Do you mean this text? "(chapters started before this have full email accounts, but due to the overhead and changes by our hosting provider, new emails addresses can only be a regular email -or- a forwarding email, not both)"
3. Yes, good tip.

Reply:
2. yes!